examples/saveload.rs

```diff
@@ -9,6 +9,10 @@ use specs::error::NoError;
 use specs::prelude::*;
 use specs::saveload::{DeserializeComponents, MarkedBuilder, SerializeComponents, U64Marker, U64MarkerAllocator};
 
+// This is an example of how the serialized data of two entities might look on disk.
+//
+// When serializing entities, they are written in an array of unnamed structs, each struct representing one entity.
```

Comment:
unnamed structs -> tuples

That's only if you pass a tuple of component storages though. Custom structs are possible, and will be easier with the derive PR.